docs/deployment/rollback.md

```diff
@@ -0,0 +1,40 @@
+# Making a rollback release
+
+This list outlines the manual steps needed to make a rollback of the
+`benefits` app.
+
+If a change is deployed to the app that makes it fail to start, making a rollback
+will deploy the app to a known working state again.
+
+[Start a new Release on Github](https://github.com/cal-itp/benefits/issues/new?labels=release&template=release.yml&title=Make+a+Release){ .md-button }
+
+## 0. Create a temporary branch from the last known good commit
+
+```bash
+git checkout -b <rollback-branch> <commit-hash>
+```
+
+Replace `<rollback-branch>` with the rollback branch name and `<commit-hash>` with the last known good commit hash.
+
+## 1. Tag the HEAD of the rollback branch with a release tag
+
+```bash
+git tag -a YYYY.0M.R
+```
+
+Git will open your default text editor and prompt you for the tag annotation. For the tag annotation,
+use the release tag version and close the text editor.
+
+## 2. Push the tag to GitHub to kick off the rollback
+
+```bash
+git push origin YYYY.0M.R
+```
+
+## 3. Fix whatever issue is wrong using the rollback branch
+
+Commit the code changes that fix the issue that prompted the rollback.
```

Comment:
Same question as above, with full question written out in https://github.com/cal-itp/benefits/issues/1249#issuecomment-2270038816

Reply:
Yep, I updated the steps in 66ac896dd7d86706aec826e48adf7ae6dfd49384 to match the rollback checklist.